services/app-api/forms/sar.json

```diff
@@ -30,9 +163,58 @@
           "pageType": "standard",
           "verbiage": {
             "intro": {
-              "section": "",
-              "subsection": ""
-            }
+              "section": "Recruitment, Enrollment, and Transitions",
+              "subsection": "Number of people who signed an MFP informed consent form in the reporting period",
+              "hint": "In this section, provide information for the specified period.",
+              "info": [
+                {
+                  "type": "h3",
+                  "content": "Number of people who signed an MFP informed consent form in the {January 1 to June 30, 2023} reporting period"
+                },
+                {
+                  "type": "span",
+                  "content": "Number of institutional residents who have signed an informed consent form indicating their desire to transition to the community and enroll in the state's MFP program."
+                }
+              ]
+            },
+            "reviewPdfHint": "To view totals, click <i>Review PDF</i> and it will open a summary in a new tab."
+          },
+          "form": {
+            "id": "ret-psmicf",
+            "fields": [
+              {
+                "id": "ret_numberOfOlderAdults",
+                "type": "text",
+                "validation": "text",
+                "props": {
+                  "label": "Number of Older Adults"
+                }
+              },
+              {
+                "id": "ret_numberOfPD",
+                "type": "text",
+                "validation": "text",
+                "props": {
+                  "label": "Number of PD"
+                }
+              },
+              {
+                "id": "ret_numberOfIDD",
+                "type": "text",
+                "validation": "text",
+                "props": {
+                  "label": "Number of I/DD"
+                }
+              },
+              {
+                "id": "ret_numberOfMHSUD",
+                "type": "text",
+                "validation": "text",
```

Comment:
do we want these to be number validation types?

Reply:
ooh, good point!